src/base/connectionView.js

```diff
@@ -435,7 +437,7 @@ class ConnectionView extends BaseView {
     let j;
 
     // We need the highest RPS connection to make this volume relative against
-    if (!this.object.volumeGreatest) { return; }
+    if (!this.object.volumeGreatest && this.object.volumeGreatest !== 0) { return; }
```

Comment:
What are you trying to check for with this additional condition?
The condition will never hit anyways. if `this.object.volumeGreatest` exists, it will fail before checking the second condition.